lib/tasks/paperclip.rake

```diff
@@ -109,7 +123,20 @@ namespace :paperclip do
     end
   end
 
- desc "find missing attachments. Useful to know which attachments are broken"
+  desc "Upgrades the MD5 fingerprints to the SHA1 fingerprints."
```

Comment:
Do you mean `SHA256` instead of `SHA1`? In any case, where do you set the option?